src/main/java/frc/robot/RobotContainer.java

```diff
@@ -127,10 +128,13 @@ public RobotContainer() {
   }
 
   private void configureAuto() {
-    autoChooser = new SendableChooser<>();
-    /* This is where you put auto commands. */
-
+    /* Build an auto chooser. This will use Commands.none() as the default option. */
+    autoChooser = AutoBuilder.buildAutoChooser();
+    /* Another option that allows you to specify the default auto by its name */
+    /* autoChooser = AutoBuilder.buildAutoChooser("My Default Auto"); */
     SmartDashboard.putData("choose auto", autoChooser);
+
+    /* This is where you put auto commands. */
```

Comment:
Move this comment above the `SmartDashboard.putData` call, since we'll need to add auto commands *before* we send the chooser to the SmartDash. 

Also probably clarify the comment to specify that to add autos, you'll need to call `autoChooser.addOption`